loss-detection-app/pipelines/object_detection/3/pipeline.json

```diff
@@ -1,13 +1,11 @@
 {
-	"name": "object_detection",
-	"version": 1,
 	"type": "GStreamer",
-	"template": "urisourcebin uri=\"{source[uri]}\" ! queue ! decodebin ! videoconvert name=\"videoconvert\" ! video/x-raw ! videocrop name=\"videocrop\" ! queue leaky=upstream ! gvadetect device=CPU inference-id=inf0 model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvametaconvert converter=json method=detection source=\"{source[uri]}\" name=\"jsonmetaconvert\" include-no-detections=true ! queue ! gvametapublish method=\"{destination[type]}\" address=\"{destination[host]}\" topic=\"{destination[topic]}\" timeout=1000 ! gvawatermark ! queue ! videoconvert ! fpsdisplaysink video-sink=xvimagesink sync=false",
+	"template": "urisourcebin uri=\"{source[uri]}\" ! queue ! decodebin ! videoconvert name=\"videoconvert\" ! video/x-raw ! videocrop name=\"videocrop\" ! queue leaky=upstream ! gvadetect device=CPU model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvametaconvert name=\"metaconvert\" source=\"{source[uri]}\" add-empty-results=true ! queue ! gvametapublish name=\"destination\" ! gvawatermark ! queue ! videoconvert ! fpsdisplaysink video-sink=xvimagesink sync=false",
```

Comment:
See above about source handling in urisourcebin and metaconvert

Reply:
Done